scripts/jupyterhub-singleuser

```diff
@@ -171,7 +171,7 @@ class SingleUserNotebookApp(NotebookApp):
     hub_api_url = Unicode(config=True)
     aliases = aliases
     flags = flags
-    open_browser = False
+    open_browser = Bool(False, config=True, help='Here for backwards compatible reasons. Leave as False')
```

Comment:
@robnagler Perhaps on the help phrase: `help='Leave parameter set as False. Only change to maintain backwards compatibility.'